rclcpp_components/CMakeLists.txt

```diff
@@ -73,12 +73,21 @@ if(BUILD_TESTING)
   set(components "${components}test_rclcpp_components::TestComponentFoo;$<TARGET_FILE:test_component>\n")
   set(components "${components}test_rclcpp_components::TestComponentBar;$<TARGET_FILE:test_component>\n")
   set(components "${components}test_rclcpp_components::TestComponentNoNode;$<TARGET_FILE:test_component>\n")
+  set(components "${components}test_rclcpp_components::TestComponentThrows;$<TARGET_FILE:test_component>\n")
+
+  # Used to catch invalid resource entry
+  set(invalid_components "test_rclcpp_components::TestComponentFoo;;$<TARGET_FILE:test_component>\n")
```

Comment:
it's the double ";" need it here or only just one?

Reply:
This is supposed to be improperly formed. This is checked in https://github.com/ros2/rclcpp/blob/85193e60c7b0fcd88f3bcb09d9167d2c424cbb73/rclcpp_components/test/test_component_manager.cpp#L96

I added more to the comment above so it's a bit more clear.